lib/rubocop/cop/graphql/argument_uniqueness.rb

```diff
@@ -93,6 +93,7 @@ def field_name(node)
           "`#{first_argument.value}`"
         end
 
+        # @!method argument_declarations(node)
```

Comment:
what does it do?

Reply:
This is an autocorrected by `InternalAffairs/NodeMatcherDirective`.

>https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/InternalAffairs/NodeMatcherDirective
>Checks that node matcher definitions are tagged with a YARD `@!method` directive so that editors are able to find the dynamically defined method.